recommender/engine/preprocessing/preprocessing.py

```diff
@@ -200,32 +200,30 @@ def user_and_services_to_tensors(user, services):
     and compose them into tensors ready for inference.
```

Comment:
In few places we still are - it is in this state for now because I didn't want to refactor whole project too much. After merging it will be refactored.